source/security/key-management.rst

```diff
@@ -39,3 +39,63 @@ Cryptographic keys used for signing |SPN| binaries should be securely managed.
 A leak of private key would allow a hacker to install/push compromised |SPN| or rootkits on platforms.
 Here is a `link <https://csrc.nist.gov/CSRC/media/Publications/white-paper/2018/01/26/security-considerations-for-code-signing/final/documents/security-considerations-for-code-signing.pdf>`_ to NIST whitepaper which provides BKM's for Code signing and secure key management.
 
+

```

Comment:
Can you add a couple of lines providing the context at the top of this page?

Reply:
Done

Reply:
Updated as recommended